R/model.R

```diff
@@ -722,113 +722,9 @@ CmdStanModel$set("public", name = "check_syntax", value = check_syntax_method)
 #'   ```
 #'
 #' @template model-common-args
-#' @section Arguments unique to the `sample` method: In addition to the
-#'   arguments above, the `$sample()` method also has its own set of arguments.
-#'
-#'   The following three arguments are offered by CmdStanR but do not correspond
-#'   to arguments in CmdStan:
-#'
-#'   * `chains`: (positive integer) The number of Markov chains to run. The
-#'   default is 4.
-#'
-#'   * `parallel_chains`: (positive integer) The _maximum_ number of MCMC chains
-#'   to run in parallel. If `parallel_chains` is not specified then the default
-#'   is to look for the option `"mc.cores"`, which can be set for an entire \R
-#'   session by `options(mc.cores=value)`. If the `"mc.cores"` option has not
-#'   been set then the default is `1`.
+#' @template model-sample-args
```

Comment:
I moved all the arguments unique to the sample method into a separate roxygen2 template. That way sample and sample_mpi can share them.